cmd/runner/main.go

```diff
@@ -356,28 +358,46 @@ func countCSVLines(path string) (int, error) {
 	return lineCount - 1, nil
 }
 
+func createAuditReq() (*task.CreateAuditRequest, error) {
+	car := os.Getenv("CREATE_AUDIT_REQUEST")
+	if car == "" {
+		return nil, errors.New("no CREATE_AUDIT_REQUEST was given")
+	}
+	var task task.CreateAuditRequest
+	if err := json.NewDecoder(strings.NewReader(car)).Decode(&task); err != nil {
+		return nil, fmt.Errorf("failed to load CreateAuditRequest: %w", err)
+	}
+	return &task, nil
+}
+
 func createReportReq() (*task.CreateReportRequest, error) {
-	pID := os.Getenv("PORTFOLIO_ID")
-	if pID == "" {
-		return nil, errors.New("no PORTFOLIO_ID was given")
+	crr := os.Getenv("CREATE_REPORT_REQUEST")
+	if crr == "" {
+		return nil, errors.New("no CREATE_REPORT_REQUEST was given")
+	}
+	var task task.CreateReportRequest
+	if err := json.NewDecoder(strings.NewReader(crr)).Decode(&task); err != nil {
+		return nil, fmt.Errorf("failed to load CreateReportRequest: %w", err)
 	}
+	return &task, nil
+}
 
-	return &task.CreateReportRequest{
-		PortfolioID: pacta.PortfolioID(pID),
-	}, nil
+func (h *handler) createAudit(ctx context.Context, taskID task.ID, req *task.CreateAuditRequest) error {
+	return fmt.Errorf("not implemented")
```

Comment:
niiit

```suggestion
	return errors.New("not implemented")
```

Normally `nogo` should pick stuff like this up, have we disabled it on this project?

Reply:
I don't think nogo is set up for this project. I didn't find top level references to it. I'll add a Bug to track adding it.

Reply:
#115